pkg/user/manager.go

```diff
@@ -6,16 +6,31 @@ import (
 	apitypes "k8s.io/apimachinery/pkg/types"
 )
 
+type TokenInput struct {
+	TokenName    string
+	Description  string
+	Kind         string
+	UserName     string
+	AuthProvider string
+	Ttl          *int64
+	Randomize    bool
+}
+
+type ClusterTokenInput struct {
```

Comment:
I think we should drop this type and have a clusterName param, seems cleaner vs wrapping the object above when needing to call `EnsureClusterToken`